Jenkinsfile

```diff
@@ -0,0 +1,63 @@
+#!groovy
+
+pipeline {
+  agent none
+
+  options {
+    disableConcurrentBuilds()
+  }
+
+  stages {
+    stage('Build Docker image') {
+      agent any
+      steps {
+        script {
+          def dockerRepoName = 'zooniverse/cellect_ex'
```

Comment:
This might be a good opportunity to change the repo name to designator.

Reply:
I had that thought too but I wasn't certain what other downstream effects there might be.